.github/workflows/on_pull_request.yaml

```diff
@@ -3,4 +3,4 @@ on:
 
 jobs:
```

Comment:
Are we intentionally not running integration tests here?

Reply:
Yes for the moment, since the zookeeper version (3.6.4)  in the current image is different from the one used in the charm (3.6.3) and so the test will work. When we will use the new image in the charm I will enable the test.
For the moment I manually tested the image on a temporary branch and all test passed.

Reply:
I would side on updating the CI to run those tests anyways and choosing to ignore them for a PR merge if thats the case.  The whole point of the CI here is to be predictive so its okay if they fail.  The benefit is testing beforehand not necessarily them passing beforehand.